### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by rlin0**
### Summary
- combined arc and bod into 1 feedback module

### CODE REVIEW

File: donut/modules/feedback/routes.py [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+    if data['email']:
+        helpers.add_email(group, complaint_id, data['email'])
+    return flask.redirect(
+        flask.url_for('feedback.feedback_view_complaint', group=group, id=id))
+
+
+# Remove an email from this complaint
+@blueprint.route('/1/feedback/<group>/removeEmail/<id>', methods=['POST'])
+def feedback_remove_email(group, id):
+    if group not in Groups:
```
Comment (koivyr) [inline donut/modules/feedback/routes.py:161]: You repeat a lot of the status checking conditions-- maybe roll them into a helper function?

File: donut/modules/feedback/routes.py [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+        flask.abort(400)
+        return
+    return 'Success'
+
+
+# Add an email to this complaint
+@blueprint.route('/1/feedback/<group>/addEmail/<id>', methods=['POST'])
+def feedback_add_email(group, id):
+    if group not in Groups:
+        return flask.render_template("404.html")
```
Comment (koivyr) [inline donut/modules/feedback/routes.py]: can just do ```flask.abort(404)``` here and everywhere else

File: donut/modules/feedback/routes.py [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+
+# Mark a complaint unread
+@blueprint.route('/1/feedback/<group>/markUnread/<id>')
+def feedback_mark_unread(group, id):
+    if group not in Groups:
+        return flask.render_template("404.html")
+    # Authenticate
+    if 'username' not in flask.session or not auth_utils.check_permission(
+            flask.session['username'], permissions[group].TOGGLE_READ):
+        flask.abort(403)
```
Comment (koivyr) [inline donut/modules/feedback/routes.py]: ```return flask.abort(403)``` here and everywhere else.

File: donut/modules/feedback/email_templates.py [hunk 4]
```diff
@@ -0,0 +1,7 @@
+added_message = \
+"""
+We have received BoD feedback that you're subscribed to.
```
Comment (koivyr) [inline donut/modules/feedback/email_templates.py]: Change BoD to ```{}```

File: donut/modules/feedback/routes.py (hunk 1 above)
Comment (rlin0) [reply donut/modules/feedback/routes.py:161]: They're different permissions though. I feel like it's not worth?

File: donut/modules/feedback/permissions.py [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+class BOD_PERMISSIONS(enum.IntEnum):
+    # view the summary page (BoD members only)
+    SUMMARY = 8
+    TOGGLE_READ = 9
+    ADD_REMOVE_EMAIL = 10
+    VIEW_EMAILS = 11
+
+
+class ARC_PERMISSIONS(enum.IntEnum):
+    SUMMARY = 12
```
Comment (calebsander) [inline donut/modules/feedback/permissions.py]: I think permission 12 is now used for masquerading, so we should pick a different number. Are you sure these are all in the database? I thought Logan looked for an unused permission number

File: donut/templates/includes/header.html [hunk 6]
```diff
@@ -20,19 +20,21 @@
                             <li><a href=" {{ url_for('campus_positions') }}">Campus Positions</a></li>
                             <li><a href="#">Clubs</a></li>
                             <li><a href=" {{ url_for('contact') }} ">Contacts</a></li>
+                            <li><a href="#">Documents</a></li>
```
Comment (calebsander) [inline donut/templates/includes/header.html]: Documents link is duplicated

**Review by calebsander — CHANGES_REQUESTED** [review]
Looks fine; just a few comments relating to merging with other recent changes

**Review by koivyr — DISMISSED** [review]
Just do Caleb's feedback^

**Review by calebsander — APPROVED** [review]

**Pull request merged**